lib/build.js

```diff
@@ -294,13 +294,14 @@ export default {
           success = success && !this.errorMatcher.hasMatch();
         }
 
+        const path = require('path');
         this.linter && this.linter.setMessages(this.errorMatcher.getMatches().map(match => ({
           type: 'Error',
           text: match.message || 'Error from build',
-          filePath: match.file,
+          filePath: path.isAbsolute(match.file) ? match.file : path.join(atom.project.getPaths()[0], match.file),
           range: [
             [ (match.line || 1) - 1, (match.col || 1) - 1 ],
-            [ (match.line || 1) - 1, (match.col || 1) - 1]
+            [ (match.line_end || match.line || 1) - 1, (match.col_end || match.col || 1) - 1 ]
```

Comment:
Nice, could you also add this to the README.

Now that I am seeing this, the whole linter integration thingie should probably have been in `error-matcher.js` insteaed. Not that that has anything to do with your PR :)